### PR TITLE
adds licensing information

### LINKED ISSUE #20: License for this repo?
Work is all rights reserved unless otherwise specified; I think we want to specify a more open license for this work. I would suggest CC0, MIT, or Apache 2.0 (CC0 as my initial inclination).

Thoughts @b5 @Mr0grog @lightandluck @osterbit?

### PR DESCRIPTION
**Pull request by Frijol**
fixes https://github.com/edgi-govdata-archiving/eis-search/issues/20

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,3 +40,12 @@ This app makes an archive of the EPA's Environmental Impact Statements (EIS) sea
...
 5. Server is running at http://localhost:8094 (or other port as specified in command line output)
+
+## License & Copyright
+
+Copyright (C) 2019 Environmental Data and Governance Initiative (EDGI)
+This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, version 3.0.
+
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+See the [`LICENSE`](/LICENSE) file for details.
```
Comment (Mr0grog) [inline README.md:51]: I *think* this renders fine most everywhere, but I usually use a relative link to make sure that’s always the case:

```suggestion
See the [`LICENSE`](./LICENSE) file for details.
```

**Review by Mr0grog — APPROVED** [review]
Looks good to me! Made one small note inline, but I’m not entirely sure if it actually matters.

File: README.md (hunk 1 above)
Comment (Mr0grog) [reply README.md:51]: Oh, huh, I guess this is how we have it written (non-relative) in the example from overview repo. idunno ¯\\\_(ツ)\_/¯

Comment (Frijol) [reply README.md:51]: Yeah I'm often confused about relative linking in GH. I've often just hard-linked so it's not confusing. I'll investigate how this does/maybe change (and in the overview too if needed)

Comment (Frijol) [reply README.md:51]: In the file explorer, it seems to work as is. Gonna merge and see what happens!

**Pull request merged**